<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DictDialog</class>
 <widget class="QDialog" name="DictDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择默写内容</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择课本</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择课程</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>您选择的是：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="BookCBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>104</width>
     <height>41</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="LessonCBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DictDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DictDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
